shiny/render/transformer/_transformer.py

```diff
@@ -800,33 +803,3 @@ async def resolve_value_fn(value_fn: ValueFn[IT]) -> IT:
 
 
 R = TypeVar("R")
-
-
-def decorator_args_passthrough(
```

Comment:
This was an unused function, from an older implementation that I moved away from after @wch code review feedback.